### ISSUE TITLE
关键信息抽取max_seq_len

### ISSUE THREAD

**Issue by azhaoid** [PaddlePaddle/PaddleOCR#9897]
关键信息提取的max_seq_len长度最长只有514吗？不能超出吗？
运行命令：python tools/train.py -c configs/kie/vi_layoutxlm/ser_vi_layoutxlm_xfund_zh.yml

报错信息：
alueError: (InvalidArgument) The value (514) of the non-singleton dimension does not match the corresponding value (1024) in shape for expand_v2 op.
  [Hint: Expected vec_in_dims[i] == expand_shape[i], but received vec_in_dims[i]:514 != expand_shape[i]:1024.] (at /paddle/paddle/phi/kernels/impl/expand_kernel_impl.h:61)

配置文件：configs/kie/vi_layoutxlm/ser_vi_layoutxlm_xfund_zh.yml
Train:
  dataset:
    name: SimpleDataSet
    data_dir: 
    label_file_list: 
      - 
    ratio_list: [ 1.0 ]
    transforms:
      - DecodeImage: # load image
          img_mode: RGB
          channel_first: False
      - VQATokenLabelEncode: # Class handling label
          contains_re: False
          algorithm: *algorithm
          class_path: *class_path
          use_textline_bbox_info: &use_textline_bbox_info True
          # one of [None, "tb-yx"]
          order_method: &order_method "tb-yx"
      - VQATokenPad:
          max_seq_len: &max_seq_len 512
          return_attention_mask: True
      - VQASerTokenChunk:
          max_seq_len: *max_seq_len
      - Resize:
          size: [448,448]
      - NormalizeImage:
          scale: 1
          mean: [ 123.675, 116.28, 103.53 ]
          std: [ 58.395, 57.12, 57.375 ]
          order: 'hwc'
      - ToCHWImage:

**Comment by andyjpaddle** [comment]
运行命令是啥？

**Comment by azhaoid** [comment]
> 运行命令是啥？

python tools/train.py -c configs/kie/vi_layoutxlm/ser_vi_layoutxlm_xfund_zh.yml

**Comment by azhaoid** [comment]
> 运行命令是啥？

可以增加这个长度到1024吗？

**Comment by andyjpaddle** [comment]
使用的是默认配置吗，训练数据有的是哪个呢？

**Comment by azhaoid** [comment]
> 使用的是默认配置吗，训练数据有的是哪个呢？
是用的自己的数据训练。大部分用的默认配置，只是想把下面这个参数改成1024.就报错了。
- VQATokenPad:
max_seq_len: &max_seq_len 1024

**Comment by azhaoid** [comment]
> 使用的是默认配置吗，训练数据有的是哪个呢？

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/42445886/2b728ac8-b821-4e60-a0e3-f6095aa6a3bf)
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/42445886/2de90cb9-abbe-4e8c-af71-5ccf8beab77f)

**Comment by azhaoid** [comment]
> 使用的是默认配置吗，训练数据有的是哪个呢？

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/42445886/5cb31759-0c11-4983-9992-d85dd4825b59)
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/42445886/df1fe10f-8ab3-4646-8b0f-917393d02605)

**Comment by andyjpaddle** [comment]
目前不支持修改长度

**Comment by azhaoid** [comment]
> 目前不支持修改长度

好的，谢谢。

**Comment by ariefwijaya** [comment]
@andyjiang1116 is this supported now? 

> 目前不支持修改长度
